docs/sphinx/sections/api/apidocs/resources.rst

```diff
@@ -3,9 +3,13 @@
 Resources
 =========
 
-.. autoclass:: ConfigurableResource
+Pythonic resource system
+------------------------
 
-.. autodecorator:: resource
+The following classes are used as part of the new `Pythonic resources system </concepts/resources>`_.
+
+
+.. autoclass:: ConfigurableResource
 
 .. autoclass:: ResourceDefinition
```

Comment:
I can't comment on it directly as this line didn't change, but I found this when looking at the preview:

> Resources are scoped ways to make external resources (like database connections) **available to** during job execution and to clean up after execution resolves.

Available to what? Could you please fix this?

Reply:
Fixed 👍